CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-## Changelog
+### v3.0.0-alpha0 (18-May-2017)
+
+Storybook 3.0 is our first fully community-driven release! Notable changes:
+- Move from `@kadira` to `@storybooks` org across [github](https://github.com/storybooks/storybook/), [npm](FIXME), [docs](https://storybooks.js.org/)
+- Upgrade to Webpack2 [PR637](https://github.com/storybooks/storybook/pull/637)
```

Comment:
@apexskier I just used the  PR format since it was already setup that way. I have converted to using normal links, although they don't seem to be collapsing properly (perhaps because I'm viewing them on a fork of the main repo???) Further suggestions much appreciated!